docs/editors-guide/quality-control-tests.md

```diff
@@ -341,16 +339,12 @@ WHERE
       "OMIM",
       "OMIMPS",
       "OMIA",
-      "OMOP",
       "ONCOTREE",
       "Orphanet",
-      "PATO",
       "PMID",
-      "Reactome",
       "SCDO",
       "SCTID",
       "UMLS",
-      "Wikidata",
```

Comment:
Why are the resources above removed from the qc-illegal-prefix-on-xref-annotation query?